extensions/replication/replicationStatusProcessor/ReplicationStatusProcessor.js

```diff
@@ -104,7 +104,12 @@ class ReplicationStatusProcessor {
             queueProcessor: this.processKafkaEntry.bind(this),
             bootstrap: options && options.bootstrap,
         });
-        this._consumer.on('error', () => {});
+        this._consumer.on('error', () => {
+            if (!this._consumer._consumerReady) {
```

Comment:
@jonathan-gramain Is it better like this?

Reply:
Actually I think it's not a good idea to access the private members directly.

Can you instead use a local variable that you update when the `'ready'` event has been emitted?